### PR TITLE
Removing serialport instructions

### PR DESCRIPTION
**Pull request by munyirik**

### CODE REVIEW

File: en-US/win10/samples/NodejsCylon.md [hunk 1]
```diff
@@ -66,19 +66,6 @@ Cylon.robot({
 * Run `npm install cylon cylon-firmata cylon-gpio cylon-i2c`
 
 
-### Get Serialport
-**Note:** Even though serialport is installed when Cylon is installed, you still need to get a version that:
```
Comment (zhuridartem) [inline en-US/win10/samples/NodejsCylon.md:70]: Consider the "> **Note:** markup

File: en-US/win10/samples/NodejsCylon.md [hunk 2]
```diff
@@ -66,19 +66,6 @@ Cylon.robot({
 * Run `npm install cylon cylon-firmata cylon-gpio cylon-i2c`
 
 
-### Get Serialport
```
Comment (munyirik) [inline en-US/win10/samples/NodejsCylon.md:69]: Although 'get serialport' instructions are no longer needed I should replace it with a step to build serialport for arm which is not obvious. Will make the update.

File: en-US/win10/samples/NodejsCylon.md (hunk 1 above)
Comment (munyirik) [inline en-US/win10/samples/NodejsCylon.md:70]: I was removing this section.

Comment (munyirik) [inline en-US/win10/samples/NodejsCylon.md:70]: But noted for future updates:)

**Comment by ivorb** [comment]
@munyirik Should I wait for another updated for how to build serialport, or would you like this merged now?

**Comment by munyirik** [comment]
@ivorb this can be merged now.

**Pull request merged**